src/api/layers/ExternalLayer.class.js

```diff
@@ -148,5 +156,9 @@ export default class ExternalLayer extends AbstractLayer {
             this.availableProjections.push(WGS84)
         }
         this.getFeatureInfoCapability = getFeatureInfoCapability
+        this.currentYear = currentYear
```

Comment:
does it really make sense to store this value, if we do not react/update the time config whenever it changes?

Reply:
It is used by the external layer plugin once the capabilities has been loaded, so it is required